app.js

```diff
@@ -25,18 +28,21 @@ const sessionSettings = {
 // register all partials from directory.
 hbs.registerPartials("./views/partials");
 
-app.set("views", "./views");
+app.set("views", path.join(__dirname, "views"));
```

Comment:
It should be a safe assumption that the current working directory is the root of the repo - almost every application requires it.